lib/absinthe/type/input_object.ex

```diff
@@ -70,6 +70,7 @@ defmodule Absinthe.Type.InputObject do
     fields =
       attrs
       |> Keyword.get(:fields, [])
+      |> Kernel.++(__inputname: spliced_inputname())
```

Comment:
we haven't tackled the `introspection` queries at all yet, but we might want to make sure that this field does not show up in the list, but we do kinda want it to show up as something that can auto-complete.

we probably don't need to worry about that until we get a little more traction

Reply:
noted